testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/CorruptedLogTestCase.java

```diff
@@ -0,0 +1,217 @@
+        managementClient.close();
+        container.stop(DEFAULT_FULL_JBOSSAS);
+        PathUtil.deleteRecursively(new File("target/wildfly/standalone/data/activemq/journal/attic/").toPath());
+    }
+
+    /**
+     * Set the journal file size to 100k.
+     * Send 100 messages of 20k
+     * Set the journal file size to 200k
+     * Restart the server
+     * Consume the messages
+     * Should have journal files in the attic
+     *
+     */
+    @Test
+    public void testCorruptedJournal() throws NamingException, IOException {
+        InitialContext remoteContext = createJNDIContext();
+        ConnectionFactory cf = (ConnectionFactory) remoteContext.lookup("jms/RemoteConnectionFactory");
+        Queue queue = (Queue) remoteContext.lookup("queue/corrupted");
+        try (JMSContext context = cf.createContext("guest", "guest", JMSContext.AUTO_ACKNOWLEDGE);) {
+            JMSProducer producer = context.createProducer();
+            for (int i = 0; i < 100; i++) {
+                producer.send(queue, context.createTextMessage(RandomStringUtils.randomAlphabetic(20000)));
+            }
+        }
+        setJournalSize(200L);
+        remoteContext.close();
+        managementClient.close();
+        container.stop(DEFAULT_FULL_JBOSSAS);
+        container.start(DEFAULT_FULL_JBOSSAS);
+        managementClient = createManagementClient();
+        remoteContext = createJNDIContext();
+        cf = (ConnectionFactory) remoteContext.lookup("jms/RemoteConnectionFactory");
+        queue = (Queue) remoteContext.lookup("queue/corrupted");
+        try (JMSContext context = cf.createContext("guest", "guest", JMSContext.AUTO_ACKNOWLEDGE);) {
+            JMSConsumer consumer = context.createConsumer(queue);
+            for (int i = 0; i < 100; i++) {
+                TextMessage message = (TextMessage) consumer.receive(TimeoutUtil.adjust(500));
+                Assert.assertNotNull(message);
+            }
+        }
+        Path attic = new File("target/wildfly/standalone/data/activemq/journal/attic/").toPath();
+        Assert.assertTrue("Couldn't find " + attic.toAbsolutePath(), Files.exists(attic));
+        Assert.assertTrue(Files.isDirectory(attic));
+        try (Stream<Path> stream = Files.list(attic)){
+            long nbAtticFiles = stream.collect(Collectors.counting());
+            Assert.assertTrue(nbAtticFiles > 0L);
+        }
+    }
+
+    @Test
+    public void testCorruptedJournalNotSaved() throws NamingException, IOException {
+        JMSOperations jmsOperations = JMSOperationsProvider.getInstance(managementClient.getControllerClient());
+        managementClient.getControllerClient().execute(Operations.createWriteAttributeOperation(jmsOperations.getServerAddress(), "journal-max-attic-files", 0));
+        jmsOperations.close();
+        managementClient.close();
+        container.stop(DEFAULT_FULL_JBOSSAS);
+        PathUtil.deleteRecursively(new File("target/wildfly/standalone/data/activemq/journal/attic/").toPath());
```

Comment:
Can you please parametrize this hard-coded path? Maybe that something like this could help:
```
            File target = new File(System.getProperty("jboss.home"), "standalone/data/activemq/journal/attic/");
```